frontend/src/components/smart/CharacterDisplay.vue

```diff
@@ -223,12 +224,22 @@ export default Vue.extend({
       return await this.fetchIsCharacterRaiding({ characterID });
     },
     setListClassForSelChar(id: string, currentCharId: string): any {
-      if (id === currentCharId) {
-        this.setSelectedCharacter(id);
-        return 'character-highlight';
+      const isCombatPage = (this as any).$router.currentRoute.path === '/combat';
+      const isTeamAdventureTab = (this as any).$route.query?.tab === COMBAT_TAB[COMBAT_TAB.TeamAdventure];
+      const fightMultiplier = Number(localStorage.getItem('fightMultiplier')) | 1;
+
+      if(isTeamAdventureTab && isCombatPage) {
+        const characterStamina = this.getCharacterStamina(id);
+
+        return characterStamina >= (fightMultiplier * 40) ?'character-highlight' : 'character';
```

Comment:
Here and in multiple places there seems to be a small formatting issue when it comes to spacing. Can we run some linter to fix these problems?

Reply:
I did, prolly lint ignore this as it didn't tag it as warning / error